src/s390_ecc.c

```diff
@@ -2108,6 +2119,9 @@ unsigned int eckeygen_sw(ICA_EC_KEY *key)
 	if (!EC_KEY_generate_key(a))
 		goto err;
 
+	if (!EC_KEY_check_key(a))
```

Comment:
You don't need to check the key right after it was generated.

Reply:
OK, removed.